gatsby-theme-landing-page/src/components/hero.module.css

```diff
@@ -39,6 +39,7 @@
   display: flex;
   gap: var(--space-3);
   flex-direction: column;
+  margin: 0 auto;
```

Comment:
I _think_ I added this to the base `.container` style, so you might not need this